pkg/core/client.go

```diff
@@ -65,7 +65,7 @@ type Client interface {
 }
 
 type Builder interface {
-	Build(appDir, buildImage, runImage, repoName string, log io.Writer) error
+	Build(repoName string, options BuildOptions, log io.Writer) error
```

Comment:
Builder is used to abstract and easily mock the capability of building. It should not know about the command/service type `BuildOptions` IMO

Reply:
The Builder interface is going away once we have a properly pack client API to consume. I'm not worried about it.